examples/docker/README.md

```diff
@@ -0,0 +1,20 @@
+# Docker Metrics
```

Comment:
Yeah, though the container names will be prefixed by the hostname. So if `worker-001` has Docker containers `doc-worker` and `job-runner` on it, you'll get Docker metrics that look like:

```
worker-001.docker.doc-worker.system.total
worker-001.docker.doc-worker.user_total
# etc
```

and

```
worker-001.docker.job-runner.system.total
worker-001.docker.job-runner.user_total
# etc
```

as well as `worker-001.cpu.in_use`, etc.